src/components/button/_button.scss

```diff
@@ -103,9 +109,9 @@
 
   .govuk-c-button--start {
     @include govuk-font-bold-24;
-    // TODO: Check padding values
     min-height: auto;
-    padding: em($govuk-spacing-scale-1, 24px) em($govuk-spacing-scale-6, 24px) em($govuk-spacing-scale-1, 24px) em($govuk-spacing-scale-4, 24px);
+    // Total height should equal 44px
```

Comment:
Is that because of touch targets? Would be good to specify if so.

Reply:
Aah, sorry – hadn't noticed this was the start button, so it's just bigger? In which case, should we look at increasing it to 45px to fit with the general /5px rhythm that we've established?

Reply:
Damn, yes it should... However because the font-size is not on the 5px scale we can't define equal spacing - same issue as the smaller buttons https://user-images.githubusercontent.com/23356842/33485592-d186ce34-d69e-11e7-8bb7-7511d3842b7f.png

So we'll need to make this 1px up with line-height which should distribute the 1px to 0.5px either side. However that means the unit-less line-height for start would be 1.0416666667 rather than 1